tests/unit/lms/services/user_test.py

```diff
@@ -12,17 +12,23 @@
 
 class TestUserService:
     def test_store_lti_user(
-        self, service, lti_user, db_session, application_instance_service
+        self, service, db_session, application_instance, application_instance_service
```

Comment:
This tests become a bit more complex. We need to  create a new LTIUser using the factory.